tests/compiler/test_bytecode.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+import vyper
+
+
+def test_contract_size_exceeded():
```

Comment:
This test makes an assumption about the bytecode size of a function, which could very well change in the future.  I think a better approach would be to assign or return a 24kb literal value.

Reply:
Would you suggest to assign that 24kb and then run `compile_code` or just test that passing a bytearray of that size to `build_bytecode_runtime_output` raises the warning?

Reply:
Probably better to use compileable code, in case the API changes in the future.  Just a very minimal contract that's basically..